lyx.c

```diff
@@ -1504,6 +1505,8 @@ void print_lyx_node(GString *out, node *n, scratch_pad *scratch, bool no_newline
 				g_string_append(out,"\n\n\\end_layout\n\\end_inset\n");
 			}
 			break;
+        case ADMONITION:
```

Comment:
Looks like I need to fix some spacing here to maintain consistency.  I will do that in an update to this PR